MAUI/Badge-View/Badge-customization.md

```diff
@@ -344,3 +344,46 @@ Content = sfBadgeView;
 {% endtabs %}
 
 ![BadgeView Alignment](badge-customization_images/badge_alignment.png)
+
+## Visibility of badge
+
+The IsVisible property is used to show or hide the badge in the BadgeView. By default, visibility of the badge is enabled.
```

Comment:
You can show or hide the badges in the BadgeView by setting the IsVisible property in the BadgeSetting. By default, the badge will be visible.

Reply:
Updated the description.